MemoryManagement/TypeCastsAndCStrings/TypeCastsQuestion/task.md

```diff
@@ -11,7 +11,7 @@ that can be successfully compiled without errors:
 
 ```c++
 // (1)
-double x = static_const<double>(a);
+double x = static_cast<double>(a);
```

Comment:
may I please ask you to submit various typo fixes and grammar fixes as separate PRs?